examples/TO_MIGRATE/ramses/sedo_amr.py

```
@@ -0,0 +1,255 @@
+        model.evolve_until(t)
+        current_time = t
+        plot(current_time, i)
+
+    plot(current_time, len(all_t))
+
+    # If the animation is not returned only a static image will be shown in the doc
+    ani = show_image_sequence(os.path.join(sim_folder, "rho_sedov_blast_*.png"), render_gif=True)
+
+    if shamrock.sys.world_rank() == 0:
+        # To save the animation using Pillow as a gif
+        writer = PillowWriter(fps=15, metadata=dict(artist="Me"), bitrate=1800)
```

Ruff (0.16.2)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)